### ISSUE TITLE
can not join on my server with newest version 0.4.7

### ISSUE THREAD

**Issue by MSandro** [AlexIIL/LibMultiPart#26]
```
---- Minecraft Crash Report ----
// Daisy, daisy...

Time: 3/3/21 4:27 PM
Description: Unexpected error

java.lang.IllegalStateException: [LibMultiPart] We've gotten out of sync! (Expected to be in the middle of drawing block outlines when we're not!)
	at net.minecraft.class_761.handler$ebp000$endBlockOutline(class_761.java:15422)
	at net.minecraft.class_761.method_22710(class_761.java:1333)
	at net.minecraft.class_757.method_3188(class_757.java:727)
	at net.minecraft.class_757.redirect$zlc000$redirectRenderingWorld(class_757.java:4052)
	at net.minecraft.class_757.method_3192(class_757.java:546)
	at net.minecraft.class_310.method_1523(class_310.java:1048)
	at net.minecraft.class_310.method_1514(class_310.java:681)
	at net.minecraft.client.main.Main.main(Main.java:215)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:226)
	at net.fabricmc.loader.launch.knot.Knot.init(Knot.java:139)
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:27)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at link.infra.jumploader.Jumploader.onLoad(Jumploader.java:171)
	at cpw.mods.modlauncher.TransformationServiceDecorator.onLoad(TransformationServiceDecorator.java:53)
	at cpw.mods.modlauncher.TransformationServicesHandler.lambda$loadTransformationServices$11(TransformationServicesHandler.java:131)
	at cpw.mods.modlauncher.TransformationServicesHandler$$Lambda$58/1364913072.accept(Unknown Source)
	at java.util.HashMap$Values.forEach(HashMap.java:972)
	at cpw.mods.modlauncher.TransformationServicesHandler.loadTransformationServices(TransformationServicesHandler.java:131)
	at cpw.mods.modlauncher.TransformationServicesHandler.initializeTransformationServices(TransformationServicesHandler.java:56)
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:76)
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:66)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.minecraft.class_761.handler$ebp000$endBlockOutline(class_761.java:15422)
	at net.minecraft.class_761.method_22710(class_761.java:1333)
	at net.minecraft.class_757.method_3188(class_757.java:727)
	at net.minecraft.class_757.redirect$zlc000$redirectRenderingWorld(class_757.java:4052)

-- Affected level --
Details:
	All players: 1 total; [class_746['MSandro'/2559, l='ClientWorld minecraft:overworld', x=-270.35, y=61.00, z=-1477.10]]
	Chunk stats: Client Chunks (ImmPtl) 441
	Level dimension: minecraft:overworld
	Level spawn location: World: (200,74,-1300), Chunk: (at 8,4,12 in 12,-82; contains blocks 192,0,-1312 to 207,255,-1297), Region: (0,-3; contains chunks 0,-96 to 31,-65, blocks 0,0,-1536 to 511,255,-1025)
	Level time: 73021509 game time, 30266350 day time
	Server brand: fabric
	Server type: Non-integrated multiplayer server
Stacktrace:
	at net.minecraft.class_638.method_8538(class_638.java:449)
	at net.minecraft.class_310.method_1587(class_310.java:2239)
	at net.minecraft.class_310.method_1514(class_310.java:704)
	at net.minecraft.client.main.Main.main(Main.java:215)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:226)
	at net.fabricmc.loader.launch.knot.Knot.init(Knot.java:139)
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:27)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at link.infra.jumploader.Jumploader.onLoad(Jumploader.java:171)
	at cpw.mods.modlauncher.TransformationServiceDecorator.onLoad(TransformationServiceDecorator.java:53)
	at cpw.mods.modlauncher.TransformationServicesHandler.lambda$loadTransformationServices$11(TransformationServicesHandler.java:131)
	at cpw.mods.modlauncher.TransformationServicesHandler$$Lambda$58/1364913072.accept(Unknown Source)
	at java.util.HashMap$Values.forEach(HashMap.java:972)
	at cpw.mods.modlauncher.TransformationServicesHandler.loadTransformationServices(TransformationServicesHandler.java:131)
	at cpw.mods.modlauncher.TransformationServicesHandler.initializeTransformationServices(TransformationServicesHandler.java:56)
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:76)
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:66)

-- System Details --
Details:
	Minecraft Version: 1.16.5
	Minecraft Version ID: 1.16.5
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_51, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 2250001832 bytes (2145 MB) / 5484052480 bytes (5230 MB) up to 9544663040 bytes (9102 MB)
	CPUs: 16
	JVM Flags: 5 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -Xmx10240m -Xms256m -XX:PermSize=256m
	Fabric Mods: 
		addendum: Addendum 1.16.5-1.0.2
		adorn: Adorn 1.13.0+1.16.5
		advanced_runtime_resource_pack: Runtime Resource Pack 0.3.6
		advancements-enlarger: Advancements Enlarger 0.2.3
		advdebug: Advancements Debug 2.2.0
		adventurez: AdventureZ 1.1.7
		ag4tr: Ag4tr 0.1.5.10
		allure: Allure 1.2.8
		amecsapi: Amecs API 1.1.3+mc1.16.5
		angerable-patch: Angerable Patch 1.1.0-1.16.4
		anvil_fix: AnvilFix 1.2.1-1.16
		appleskin: AppleSkin 1.0.11
		appliedenergistics2: Applied Energistics 2 8.2.0-alpha.2
		aquarius: Aquarius 1.0.3
		architectury: Architectury 1.8.130
		arstheurgia: Ars Theurgia 1.0.6
		artifacts: Artifacts 2.3.0+fabric
		artifice: Artifice 0.14.6+1.16.5
		artofalchemy: Art of Alchemy: Memoriam 1.0.0-rc4+1.16.4
		astromine: Astromine 1.11.5+fabric-1.16.5
		astromine-core: Astromine: Core 1.11.5+fabric-1.16.5
		astromine-discoveries: Astromine: Discoveries 1.11.5+fabric-1.16.5
		astromine-foundations: Astromine: Foundations 1.11.5+fabric-1.16.5
		astromine-technologies: Astromine: Technologies 1.11.5+fabric-1.16.5
		astromine-transportations: Astromine: Transportations 1.11.5+fabric-1.16.5
		attributefix: Attribute Fix {FABRIC} 1.0.1
		audiooutput: AudioOutput 0.0.3
		authme: Auth Me 1.4.0
		autoconfig1u: Auto Config v1 Updated 3.3.1
		automated_crafting: Automated Crafting 1.3.21+MC1.16.2-1.16.5
		automotons: Automotons 1.0.1
		bambootweaks: Bamboo Tweaks unspecified
		battletowers: Battle Towers 3.3.3-1.16.4
		bebooks: Better Enchanted Books 1.2.5
		bedspreads: Bedspreads 1.3.1-1.16.4
		beebetter: Bee Better 1.8.0-1.16.4
		bettercaves: YUNG's Better Caves 1.16.5-1.1
		betterend: Better End 0.8.8-beta
		bettermineshafts: YUNG's Better Mineshafts 1.16.4-1.0
		biome_locator: Biome Locator 1.1.2
		biomemakeover: Biome Makeover 1.16.4-1.2.11
		bitsandchisels: BitsAndChisels 2.1.2
		blade: Blade 2.0.0
		blast: BLAST 1.5
		blockshifter: Blockshifter 1.3.2+fabric-1.16.4
		blockus: Blockus 2.1.0+1.16.5
		blue_endless_jankson: jankson 1.2.0
		bno: Basic Nether Ores 1.16.5-5.5.0-Fabric
		boringbackgrounds: Boring Backgrounds 3.0.2+1.16.3
		byg: BYG 1.1.4
		campanion: Campanion 1.3.5
		can-i-mine-this-block: cAn i MiNe thIS bLOCk? 1.0.2
		cardinal-components: Cardinal Components API 2.7.12
		cardinal-components-base: Cardinal Components API (base) 2.7.12
		cardinal-components-block: Cardinal Components API (blocks) 2.7.12
		cardinal-components-chunk: Cardinal Components API (chunks) 2.7.12
		cardinal-components-entity: Cardinal Components API (entities) 2.7.12
		cardinal-components-item: Cardinal Components API (items) 2.7.12
		cardinal-components-level: Cardinal Components API (world saves) 2.7.12
		cardinal-components-scoreboard: Cardinal Components API (scoreboard) 2.7.12
		cardinal-components-util: Cardinal Components API (utilities) 2.7.12
		cardinal-components-world: Cardinal Components API (worlds) 2.7.12
		carrier: Carrier 1.7.4
		cavebiomes: Cave Biomes 0.6.2
		ce_foodstuffs: Common Expansion: Foodstuffs 1.1.3
		chainmail: Chainmail 0.5.0
		chains: Chains-Link 1.16.2-2.0.1
		charm: Charm 2.2.2
		charmonium: Charmonium 2.3.2
		cherishedworlds: Cherished Worlds 1.3.0-1.16.4
		chunkpregen: Fabric Chunk Pregenerator 0.3.3
		cinderscapes: Cinderscapes 1.3.3
		cleancut: CleanCut mc1.16-2.1-fabric
		cloth-api: Cloth API 1.4.9
		cloth-armor-api-v1: Cloth Armor API v1 1.4.9
		cloth-basic-math: Cloth Basic Math 0.5.1
		cloth-client-events-v0: Cloth Client Events v0 1.5.47
		cloth-common-events-v1: Cloth Common Events v1 1.4.9
		cloth-config2: Cloth Config v4 4.11.14
		cloth-datagen-api-v1: Cloth Datagen v1 1.4.9
		cloth-durability-bar-api-v1: Cloth Durability Bar API v1 1.5.47
		cloth-dynamic-registry-api-v1: Cloth Dynamic Registry API v1 1.4.9
		cloth-scissors-api-v1: Cloth Scissors API v1 1.4.9
		cloth-utils-v1: Cloth Utils v1 1.4.9
		com_electronwill_night-config_core: core 3.6.2
		com_electronwill_night-config_toml: toml 3.6.2
		com_github_nathanpb_safer: Safer 1.1.0
		com_github_therandomlabs_trlutils-config: TRLUtils-Config master-SNAPSHOT
		com_github_therandomlabs_trlutils-fabric: TRLUtils-Fabric 1.16-SNAPSHOT
		com_moandjiezana_toml_toml4j: toml4j 0.7.2
		com_udojava_evalex: EvalEx 2.5
		completeconfig: CompleteConfig 0.12.0
		compressedblocks: Compressed Blocks 1.0
		computercraft: CC:T for Fabric 1.91.2
		conjuring: Conjuring 0.1.4
		controlling: Controlling For Fabric 1.1.2
		cotton: Cotton 1.1.0
		cotton-cauldron: Cotton Cauldron 1.1.0
		cotton-commons: Cotton Commons 1.0.4
		cotton-config: Cotton Config 1.0.0-rc.7
		cotton-datapack: Cotton Datapack 1.0.7
		cotton-logging: Cotton Logging 1.0.0-rc.4
		cotton-player-events: Cotton Player Events 1.0.2
		couplings: Couplings 1.3.3
		crookedcrooks: Crooked Crooks 1.3.3+1.16.4
		croptopia: Croptopia 1.2.0
		croptosis: Croptosis 1.0.1
		crowdin-translate: CrowdinTranslate 1.2
		crumbs: Crumbs 1.0.6
		crusade: Crusade 1.2.0
		cullleaves: Cull Leaves 2.1.0
		damagetilt: Damage Tilt 1.16-fabric-0.1.2
		dankstorage: Dank Storage 1.9a-1.16.3
		darkpaintings: Dark Paintings 3.0.1
		dawn: Dawn API 1.7
		daylightdimension: Daylight Dimension 0.0.2
		decimal_coins: Decimal Coins 1.16.2-1.2.0
		decorative: Decorative 4.0.0
		dehydration: Dehydration 1.0.9
		diggusmaximus: Diggus Maximus 1.4.2-1.16.3
		disable_custom_worlds_advice: Disable Custom Worlds Advice 1.2
		dispenser_configurator: Dispenser Configurator 1.0.1+MC1.16.4-1.16.5
		dml-refabricated: Deep Mob Learning: Refabricated 0.3.2-beta1
		doorsofinfinity: Doors of Infinity 1.0.9-1.16.4
		dynamicfps: Dynamic FPS 2.0.1
		ecotones: Ecotones 0.7.1
		enchantmentdescriptions: Enchantment Descriptions 2.0.1
		exotic-art-core: Exotic Art Core Content Pack 1.0.379
		exotic-art-tech: Exotic Art Tech Pack 1.2.379
		exotic-matter: Exotic Matter 2.12.381
		expandedstorage: ExpandedStorage 5.6.33
		extraalchemy: Extra Alchemy 1.3.1
		extradoors: ExtraDoors 1.8.2-1.16.2
		ezpas: Ezpas 1.2.3-1.16.2
		fabric: Fabric API 0.31.0+1.16
		fabric-api-base: Fabric API Base 0.2.1+9354966b7d
		fabric-api-lookup-api-v1: Fabric Api Lookup API (v1) 1.0.0+8f91dfb63a
		fabric-biome-api-v1: Fabric Biome API (v1) 3.1.1+ca58154a7d
		fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.5+ca58154a7d
		fabric-command-api-v1: Fabric Command API (v1) 1.1.0+f9b3753b7d
		fabric-commands-v0: Fabric Commands (v0) 0.2.2+ca58154a7d
		fabric-containers-v0: Fabric Containers (v0) 0.1.11+9354966b7d
		fabric-content-registries-v0: Fabric Content Registries (v0) 0.2.1+ca58154a7d
		fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.1.3+ca58154a7d
		fabric-diagonal-panes: Fabric Diagonal Panes 0.4.0
		fabric-dimensions-v1: fabric-dimensions-v1 2.0.6+9354966b7d
		fabric-entity-events-v1: Fabric Entity Events (v1) 1.0.3+ca58154a7d
		fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.4.2+ca58154a7d
		fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.1+ca58154a7d
		fabric-furnaces: Fabric Furnaces 1.3.3-1.16.4
		fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.6+ca58154a7d
		fabric-item-api-v1: Fabric Item API (v1) 1.2.1+ca58154a7d
		fabric-item-groups-v0: Fabric Item Groups (v0) 0.2.3+ca58154a7d
		fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.4+9354966b7d
		fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.1+ca58154a7d
		fabric-language-kotlin: Fabric Language Kotlin 1.4.21+build.1
		fabric-language-scala: Fabric Language Scala 1.0.0
		fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 1.2.1+ca58154a7d
		fabric-loot-tables-v1: Fabric Loot Tables (v1) 1.0.2+ca58154a7d
		fabric-mining-levels-v0: Fabric Mining Levels (v0) 0.1.3+ca58154a7d
		fabric-models-v0: Fabric Models (v0) 0.2.1+ca58154a7d
		fabric-networking-api-v1: Fabric Networking API (v1) 1.0.1+ca58154a7d
		fabric-networking-blockentity-v0: Fabric Networking Block Entity (v0) 0.2.8+ca58154a7d
		fabric-networking-v0: Fabric Networking (v0) 0.3.2+ca58154a7d
		fabric-object-builder-api-v1: Fabric Object Builder API (v1) 1.9.4+9354966b7d
		fabric-object-builders-v0: Fabric Object Builders (v0) 0.7.2+ca58154a7d
		fabric-particles-v1: Fabric Particles (v1) 0.2.4+ca58154a7d
		fabric-registry-sync-v0: Fabric Registry Sync (v0) 0.7.4+ca58154a7d
		fabric-renderer-api-v1: Fabric Renderer API (v1) 0.4.1+ca58154a7d
		fabric-renderer-indigo: Fabric Renderer - Indigo 0.4.4+ca58154a7d
		fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 2.2.1+ca58154a7d
		fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.1.5+ca58154a7d
		fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 0.1.13+ca58154a7d
		fabric-rendering-v0: Fabric Rendering (v0) 1.1.2+ca58154a7d
		fabric-rendering-v1: Fabric Rendering (v1) 1.5.1+ca58154a7d
		fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.4.2+ca58154a7d
		fabric-screen-api-v1: Fabric Screen API (v1) 1.0.0+c045166c7d
		fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.1.5+9354966b7d
		fabric-structure-api-v1: Fabric Structure API (v1) 1.1.4+ca58154a7d
		fabric-tag-extensions-v0: Fabric Tag Extensions (v0) 1.1.1+ca58154a7d
		fabric-textures-v0: Fabric Textures (v0) 1.0.6+ca58154a7d
		fabric-tool-attribute-api-v1: Fabric Tool Attribute API (v1) 1.2.6+ca58154a7d
		fabricloader: Fabric Loader 0.11.1
		facility: Facility 0.12.204
		fallflyinglib: FallFlyingLib 1.1.0
		fallingtree: FallingTree 2.10.2
		fasttransferlib: Fast Transfer Lib 0.2.2
		fatxporbs: Fat Experience Orbs 0.0.9
		fermion: Fermion 2.9.218
		fermion-gui: Fermion GUI 2.5.218
		fermion-modkeys: Fermion Modifier Keys 1.10.218
		fermion-orientation: Fermion Orientation 1.2.218
		fermion-simulator: Fermion Simulator 1.8.218
		fermion-varia: Fermion Varia 2.6.218
		fiber: fiber 0.23.0-2
		fiber2cloth: Fiber To Cloth 3.0.1
		flamingo_ooo: Flamingo, oh, oh, oh... 1.0.4-MC1.16.3-fabric
		flonters: Flonters 1.1.1+1.16.2
		flora-doubling: Flora Doubling 1.3.0+1.16.4-fabric
		fluidity: Fluidity 0.16.204
		flytre_lib: FlytreLib 0.2.8
		fonthack: Font Hack 2.4.51
		framed: Framed 1.0.2
		frex: FREX 4.5.201
		gameinfo: GameInfo 2.0.1 1.16.4
		gateofbabylon: Gate Of Babylon 1.3.0-1.16.5
		geckolib3: Geckolib 3.0.23
		givemehats: Give Me Hats! 1.4
		glassdoor: Glass Doors 1.5.0-1.16
		glassential: Glassential 1.16.4-1.1.5
		go-fish: Go Fish 1.2.1-beta-1.16.4
		golemsgalore: Golems Galore 0.2.0+mc.1.16.2
		goml: Get Off My Lawn 1.3.3-beta
		grindenchantments: Grind Enchantments 1.1.3+1.16.3
		hard-hat-steve: Hard Hat Steve 1.2.0
		hardcorequesting: Hardcore Questing Mode 1.16.2+5.5.6
		harvest: Harvest 1.2.12
		harvest_scythes: Harvest Scythes 1.0.9
		hctm-base: HCTM Base 3.5.1
		healthoverlay: Health Overlay 3.6.14
		horseinfo: Horse Info 0.2.2-1.16.1
		iamverysmart: I Am Very Smart 2.1.0+mc1.16.1
		identity: Identity 1.11.9-beta-1.16.5
		ignis-fatuus: Ignis Fatuus 1.1.3
		imm_ptl_core: Immersive Portals Core 0.80
		immersive_portals: Immersive Portals 0.80
		improved-stations: Improved Stations 2.0.3
		indrev: Industrial Revolution 1.8.11-BETA
		inmis: Inmis 2.1.2-1.16.4
		inventorysorter: Inventory Sorter 1.7.7-1.16
		invisframes: Invisible Frames 2.0.1
		io_github_onyxstudios_cardinal-components-api: Cardinal-Components-API 2.7.9
		iron-jetpacks: Iron Jetpacks 0.3.1
		jankson: Jankson 3.0.1+j1.2.0
		jargm: Just Another Redstone Gate Mod 1.2.0
		java: Java HotSpot(TM) 64-Bit Server VM 8
		javax_vecmath_vecmath: vecmath 1.5.2
		json-model-extensions: JSON Model Extensions 1.21.182
		jumploader: Jumploader 2.1.2
		kibe: Kibe 1.6.4-BETA
		ktdatataglib: KtDataTagLib 1.4.0
		kubejs: KubeJS 1605.3.6-build.171
		kyrptconfig: Kytpt Config 1.1.6-1.16
		lakeside: Lakeside 1.0.3+1.16.4
		lambdabettergrass: LambdaBetterGrass 1.0.3+1.16
		lambdynlights: LambDynamicLights 1.3.4+1.16
		lamps: The Lamps! 1.1
		lapisreserve: Lapis Reserve 1.0.6
		launchersmod: Launchers 3.2.0
		libaddict: libaddict 0.2.1
		libblockattributes: LibBlockAttributes 0.8.7
		libblockattributes_core: LibBlockAttributes (Core) 0.8.7
		libblockattributes_fluids: LibBlockAttributes (Fluids) 0.8.7
		libblockattributes_items: LibBlockAttributes (Items) 0.8.7
		libcd: LibCapableData 3.0.3+1.16.3
		libgui: LibGui 3.3.5+1.16.5
		libmultipart: LibMultiPart 0.4.7
		libmultipart_deps_container: LibMultiPart (Dependency Container) 0.4.3
		libnetworkstack: Lib Network Stack 0.4.7
		library_of_exile: Library of Exile 1.0.6
		libstructure: LibStructure 1.5
		lightoverlay: Light Overlay 5.8.0
		lightstones: Lightstones 1.3.0
		lithium: Lithium 0.6.4
		magna: Magna 1.5.1-1.16.5
		maybe-data: Maybe data 1.0.0-1.16.2
		mcdict: MCDict 2.0.1+1.16.4
		mechanix: Mechanix 3.3.0
		megane: megane 3.1.3+1.16.5
		megane-api: megane-api 3.1.3+1.16.5
		megane-impl: megane-impl 3.1.3+1.16.5
		megane-runtime: megane-runtime 3.1.3+1.16.5
		mesh: Mesh 0.6.1-alpha.21+1.16.4
		mesh_api: Mesh API 0.6.1-alpha.21+1.16.4
		microconfig: Microconfig 1.0.5
		midnight-hats: Midnight Hats 2.0.1
		minecraft: Minecraft 1.16.5
		mo_glass: Mo Glass 1.5-MC1.16.5
		modern_industrialization: Modern Industrialization 0.3.14
		modmenu: Mod Menu 1.16.8
		more_tooltips: MoreTooltips 1.0.1
		moreberries: More Berries 1.3.4
		moretotems: More Totems 2.7.3
		mostructures: Mo' Structures 1.0.1+mc.1.16.4
		mousewheelie: Mouse Wheelie 1.6.4+mc1.16.4
		multi_item_lib: Multi Item Lib 1.1.1
		netherite_plus: Netherite Plus Mod 1.3.17-1.16.5
		no-potion-offset: No Potion Offset ${version}
		noanglebrackets: NoAngleBrackets 1.0.1
		nookbuild: NookBuild 1.4.0
		notes: Notes 1.0.4
		nourish: Nourish 1.2.1
		okzoomer: Ok Zoomer 4.0.1+1.16.2
		org_aperlambda_lambdajcommon: lambdajcommon 1.8.1
		org_jetbrains_annotations: annotations 20.0.0
		org_jetbrains_kotlin_kotlin-reflect: kotlin-reflect 1.4.21
		org_jetbrains_kotlin_kotlin-stdlib: kotlin-stdlib 1.4.21
		org_jetbrains_kotlin_kotlin-stdlib-jdk7: kotlin-stdlib-jdk7 1.4.21
		org_jetbrains_kotlin_kotlin-stdlib-jdk8: kotlin-stdlib-jdk8 1.4.21
		org_jetbrains_kotlinx_kotlinx-coroutines-core-jvm: kotlinx-coroutines-core-jvm 1.4.2
		org_jetbrains_kotlinx_kotlinx-coroutines-jdk8: kotlinx-coroutines-jdk8 1.4.2
		org_joml_joml: joml 1.10.0
		oro_config: OroArmor Config 1.2.0
		packages: Packages 1.1.1
		patchouli: Patchouli 1.16.4-50-FABRIC
		pehkui: Pehkui 1.8.0+21w08b
		phonos: Phonos 0.1.1-beta
		phosphor: Phosphor 0.7.1+build.11
		piglib: Piglib 1.0.1
		plantinajar: Plant In A Jar 1.8.1
		plated: Plated 2.0.1
		platos: Plato's Transporters 1.6.4
		playerabilitylib: Pal 1.2.1
		pling: Pling 1.3.0
		programmerartinjector: Programmer Art Injector 1.2.1
		qcommon-cfg-core: qcommon-cfg-core 3.2.0
		qcommon-croco: Croco 2.1.1
		quickshulker: Quick Shulker 1.1.14-1.16
		reach-entity-attributes: Reach Entity Attribute 1.0.1
		reborncore: Reborn Core 4.7.2+build.132
		redbits: Red Bits 1.7.0
		repurposed_structures: Repurposed Structures 1.16.5-1.8.3
		reroll: Reroll 1.2.0-1.16.5
		respawnablepets: Respawnable Pets 1.16.5-1
		rhino: Rhino 1.7.13-build.14
		riverredux: River Redux 0.1.6
		rocks: This Rocks! 1.3.1
		roughlyenoughitems: Roughly Enough Items 5.10.184
		roughlyenoughitems-api: REI (API) 5.10.184
		roughlyenoughitems-default-plugin: REI (Default Plugin) 5.10.184
		roughlyenoughitems-runtime: REI (Runtime) 5.10.184
		roughlyenoughresources: Roughly Enough Resources 2.1.3
		rswires: RSWires 4.10.2
		rtree-3i-lite-fabric: rtree-3i-lite for Fabric 0.3.0
		sandwichable: Sandwichable 1.2-beta5
		savemystronghold: Save My Stronghold! 1.16.4-1.0
		screenshotclipboard: Screenshot to Clipboard 1.0.6
		secretrooms: Secret Rooms 0.9.3+1.16.5
		shulkerboxtooltip: Shulker Box Tooltip 2.3.5+1.16.5
		shulkerutils: shulkerutils 1.0.1-1.16
		silky_spawners: Silky Spawners 2.1.0+build.2-1.16
		simple_pipes: SimplePipeTest 0.4.7
		simple_pipes_dep_container: SimplePipeTest (Dependency Container) 0.4.7
		simpledrawers: Simple Drawers 1.8.0-1.16.3
		simpleteleporters: Simple Teleporters 2.2.1
		simplevoidworld: Simple Void World 4.0.0
		slotlink: slotlink 2.2.0+1.16.4
		slotlock: SlotLock 1.0.1-BETA
		smithee: Smithee 0.2.3
		smootherbedrock: Smoother Bedrock 1.0.0
		someforgepatchesported: Some Forge Patches Ported 1.0.0
		soulcapture: Soul Capture 1.0.2
		spatialharvesters: Spatial Harvesters 0.11.6a
		special-circumstances: Special Circumstances 1.8.218
		spruceui: SpruceUI 2.0.4+1.16
		stacc: Stacc 1.0.1
		staffofbuilding: Staff of Building 1.2.5-1.16.5
		static-content: Static Content 1.0.1-1.16.2
		staticdata: Static Data 1.1.2
		step-height-entity-attribute: Step Height Entity Attribute 1.0.0
		stonecutter-tweaks: Stonecutter Tweaks 1.1.0-1.16
		strawdummy: Straw Dummy 1.4.0
		structure-helpers: Structure Helpers 3.0.0
		structurized: Structurized 1.1.0
		supernaturalcrops: Supernatural Crops 2.0.2
		tacocraft: TacoCraft 1.3.1+mc.1.16
		team_reborn_energy: Energy 0.1.1
		techreborn: Tech Reborn 3.8.1+build.211
		terraform-biome-builder-api-v1: Terraform Biome Builder API (v1) 1.0.4
		terraform-config-api-v1: Terraform Config API (v1) 1.0.3
		terraform-dirt-api-v1: Terraform Dirt API (v1) 1.1.3
		terraform-overworld-biome-extensions-api-v1: Terraform Overworld Biome Extensions API (v1) 1.0.3
		terraform-shapes-api-v1: Terraform Shapes API (v1) 1.0.4
		terraform-surfaces-api-v1: Terraform Surfaces API (v1) 1.0.3
		terraform-tree-api-v1: Terraform Tree API (v1) 1.0.5
		terraform-wood-api-v1: Terraform Wood API (v1) 1.0.3
		terrestria: Terrestria 2.2.0
		the_bumblezone: The Bumblezone - Fabric 1.16.5-2.3.0
		timeinabottle: Time in a Bottle 1.0.1-mc1.16.4-fabric
		tis3d: TIS-3D 1.6.0
		tis3d-additions: TIS-3D Additions 0.2.1+1.16.2
		tk_valoeghese_zoesteriaconfig: ZoesteriaConfig 1.3.2
		tooltipfix: ToolTip Fix 1.0.2-1.16
		totemmod: Wither Totems Mod 0.3.2
		transferchestmod: Transfer Chest 1.1.4
		transparent: Transparent 2.1.0
		trapexpansion: Trap Expansion 1.3.2
		traverse: Traverse 3.3.2
		trinkets: Trinkets 2.6.7
		truerooms: True Secret Rooms 1.1.3
		tweed: Tweed API 3.0.0-beta.22
		vanilla-hammers: Vanilla Hammers 3.0.2-1.16.5
		vanilla_parts: Vanilla Parts 1.1.4+1.16.3
		vanilladeathchest: VanillaDeathChest 1.16.3-1.11.0.7-fabric
		vanillaexcavators: Vanilla Excavators 1.1.0
		villagerhats: Villager Hats fabric-1.16.4-1.0.1
		voyager: Voyager 1.0.0
		waystones: Waystones 1.1.1.1
		wild_explorer: Wild Explorer 1.3.2
		winged: Winged Mod 1.16.2-1.4.1
		wither_proof_block: Wither Proof Block 1.0.3
		wolveswitharmor: Wolves With Armor 1.5.2-1.16.2
		woodenhoppers: Wooden Hoppers 1.0.0
		woods_and_mires: Woods and Mires 1.0.4+1.16.3
		world_blender: World Blender 1.16.5-2.0.2
		wrenchable: Wrenchable 1.0.0+1.16.1
		wthit: wthit 2.3.1+1.16.5
		xaerominimap: Xaero's Minimap 21.4.1
		xaeroworldmap: Xaero's World Map 1.12.1
		xb: Exotic Blocks 1.6.172
		xlpackets: XLPackets 1.1
		yalbm: Yet Another Lucky BLock Mod 1.1.9
		yosbr: YOSBR 0.1.1
		yungsapi: YUNG's API 1.16.5-Fabric-2
	Patchouli open book context: n/a
	Launched Version: Jumploader
	Backend library: LWJGL version 3.2.2 build 10
	Backend API: Radeon RX 580 Series GL version 4.6.14757 Compatibility Profile Context 20.12.1 27.20.14501.28009, ATI Technologies Inc.
	GL Caps: Using framebuffer using OpenGL 3.0
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fabric'
	Type: Client (map_client.txt)
	Graphics mode: fast
	Resource Packs: Fabric Mods, fabric-diagonal-panes/default
	Current Language: English (US)
	CPU: 16x AMD Ryzen 7 2700X Eight-Core Processor 
```

**Comment by AlexIIL** [comment]
This should be fixed in [0.5.0-pre.1](https://alexiil.uk/maven/alexiil/mc/lib/libmultipart-all/0.5.0-pre.1/libmultipart-all-0.5.0-pre.1.jar), although I'll push a proper release tomorrow.

**Comment by AlexIIL** [comment]
Okay, finally released in 0.5.0
